modules/virtual_network_gateway/variables.tf

```
@@ -108,22 +86,37 @@ variable "generation" {
 
 variable "private_ip_address_enabled" {
   description = "Should private IP be enabled on this gateway for connections?"
+  default     = false
   type        = bool
 }
 
+variable "avzones" {
```

how about chainging to `zones` ?
```suggestion
variable "zones" {
```

---

ok